pontoon/static/js/utils.js

```diff
@@ -0,0 +1,149 @@
+
+export class Cookies {
```

Comment:
well, i think its about finding a balance - i was trying to avoid installing deps, but also not re-inventing the wheel - probs in the case of cookies we could just use something - but i didnt see any nice pythonesque strip libs (at least not after a quick look)